client/me/concierge/book/info-step.js

```diff
@@ -55,7 +55,7 @@ class InfoStep extends Component {
 		return (
 			<div>
 				<PrimaryHeader />
-				<CompactCard className="concierge__site-block">
+				<CompactCard className="book__site-block">
```

Comment:
Would this classname actually make more sense if it was called `book__info-step-card` so that it matches the name of the component? Then the top `<div>` could have ` book__info-step`

Reply:
Right, updated. I thought it would be easier to share the same class for both info-step and skeleton. I think `-card` might be a bit too generic giving the fact we have multiple `CompactCard` components. This one includes only the `Site` block, so I guess `book__info-step-site-block` would be ok?